packs/vsphere/actions/hello_vsphere.py

```diff
@@ -18,5 +18,6 @@
 
 class HellowVsphere(BaseAction):
 
-    def run(self):
+    def run(self, vsphere=None):
```

Comment:
if no value is provided it looks for the old style configuration information.
If you are planning to use the new style you must provide its value.